File: task1/go/task1.go
package task1

func twoSum(nums []int, target int) []int {
	// v2
	seen := make(map[int]int)

	for i, v := range nums {
		if seenIdx, exist := seen[target-v]; exist {
			return []int{i, seenIdx}
		}

		seen[v] = i
	}

	return []int{0, 0}
}

File: task1/python/task1.py
#!python3

from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        visited = {}

        for i, v in enumerate(nums):
            if (target - v) in visited:
                return [i, visited[target - v]]
            visited[v] = i
        
        return [-1, -1]